proto/sensr_proto/output.proto

```diff
@@ -78,10 +78,16 @@ message EventMessage {
   SystemHealth health = 3;
 }
 
+message CustomMessage {
```

Comment:
Please make this in a different file for easy maintaining.

Reply:
Done!